src/shapes/sdfgrid.cpp

```diff
@@ -232,8 +237,8 @@ class SDFGrid final : public Shape<Float, Spectrum> {
 
     void traverse(TraversalCallback *callback) override {
         Base::traverse(callback);
-        callback->put_parameter("to_world",     *m_to_world.ptr(),          ParamFlags::Differentiable | ParamFlags::Discontinuous);
-        callback->put_parameter("grid",         m_grid_texture.tensor(),    ParamFlags::Differentiable | ParamFlags::Discontinuous);
+        callback->put_parameter("to_world",     *m_to_world.ptr(),          +ParamFlags::NonDifferentiable);
```

Comment:
SDFGrid is not compatible with the projective integrators?

Reply:
Yes they are, have a look at section 5.4 of [the paper](http://rgl.epfl.ch/publications/Zhang2023Projective). However, as mentioned in the paper, we did not (fully) implement a projection operation yet. 

In short, we have some code that can differentiate SDFGrids but only for certain choices of parameters in the integrators. For now, I'd prefer having no differentiation available rather than print a long wall of text explaining the few combinations of parameters that would work.